iidm/iidm-impl/src/test/java/eu/itesla_project/iidm/network/impl/LineTest.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.iidm.network.impl;
+
+import eu.itesla_project.iidm.network.Network;
+import eu.itesla_project.iidm.network.test.EurostagTutorialExample1Factory;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class LineTest {
```

Comment:
A good idea is to add a setUp() method with the "@Before" annotation to create and initialize the network.